src/Framework/Routes/Router.php

```diff
@@ -68,13 +69,21 @@ protected function getDataFromPostRequest(): array
 
         $contentType = $_SERVER['CONTENT_TYPE'];
 
-        // this content type is typically used throughout legacy with jQuery and wp-ajax
-        if (str_contains($contentType, "application/x-www-form-urlencoded")) {
-            $requestData = give_clean($_REQUEST);
+        // v2 forms use "application/x-www-form-urlencoded"
+        // v3 forms use "multipart/form-data" and/or "application/json"
+        if (str_contains($contentType, "application/x-www-form-urlencoded") || str_contains(
+                $contentType,
+                "multipart/form-data"
+            )) {
+            $requestData = array_merge(
+                give_clean($_REQUEST),
+                //TODO: check if this is necessary for validation to find the value from the request
+                give_clean($_FILES)
```

Comment:
I also plan on adding a validation rule for file upload fields which this would be necessary for to retrieve the value.